webapp/src/redux/features/app/AppState.ts

```diff
@@ -17,3 +18,31 @@ export interface Alert {
     message: string;
     type: AlertType;
 }
+
+export interface Setting {
+    title: string;
+    description?: string;
+    stackVertically?: boolean;
+    learnMoreLink?: string;
+}
+
+export interface AppState {
+    alerts: Alert[];
+    activeUserInfo?: ActiveUserInfo;
+    // Total usage across all chats by app session
+    tokenUsage: TokenUsage;
+}
+
+export const initialState: AppState = {
```

Comment:
nit: `InitialAppState`

Reply:
I believe initialState is the standard practice, so we can just pass the variable as is to createSlice rather than explicitly naming in every feature slice like this

![image](https://github.com/microsoft/chat-copilot/assets/125500434/d7587e0d-9481-4702-aef4-34ee535e28cb)

Reply:
that's fair, but if needed elsewhere it can make for confusing imports:
```ts
import { initialState } from './AppState';
import { initialState } from './ConverstationsState';
```

I don't have a strong opinion on this, although I'd say we should generally lean toward clear variable naming vs. convenience in assignment.

Reply:
We don't use initial state anywhere else, so it's well defined in the context since we follow a well-structured file naming pattern for redux features. Will leave as is